proposed/extended-coding-style-guide.md

```diff
@@ -82,9 +82,9 @@ each word is capitalised including the very first letter.
 
 ### 2.2 Files
 
-All PHP files MUST use the Unix LF (linefeed) line ending.
+All PHP files MUST use the Unix LF (linefeed) line ending, applying to every line in the file.
 
-All PHP files MUST end with a single line, containing only a single newline (LF) character.
+All PHP files MUST end with a non-blank line.
```

Comment:
This is not OK since "non-blank" isn't "empty but containing only a single newline (LF) character". Non-blank matches just a line with some code in it.